data/hai missions.txt

```diff
@@ -1061,3 +1061,93 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Enlisted by Hai [1]"
+	name "Defend <planet>"
+	description "Defeat a pirate raid on <destination>."
+	to offer
+		"combat rating" > 400
+		"random" < 35
+	source "Allhome"
+	npc
+		government "Hai"
+		personality "heroic" staying"
+		fleet "Large Hai" 2
+		fleet "Large Militia"
+	npc evade
+		government "Pirate"
+		personality "heroic" "staying"
+		fleet "Large Northern Pirates" 10
+	on offer
+		conversation
+			`Suddenly you notice a massive rush in the spaceport: "We are under attack! <planet> is under attack by the unfettered humans! We need every combat-worthy ship to join in the defenses!" The authorities will probably be very gracious, but this could also be an easy way to get yourself killed.`
+			choice
+				`	(Join the defense fleet.)`
+				`	(Stay here until the fight is over.)`
+					defer
+			`A mix of Hai and Human pilots have amassed to help repel the unfettered attack. You join them, and take off together...`
+				launch
+	on complete
+		dialog
+			`The government of <planet> pays you <payment> for helping to drive off the attack.`
+		payment 280000
+
+mission "Enlisted by Hai [2]"
+	name "Enlist with the Hai"
+	description "You have been recommended for your attentiveness for keeping the peace throughout Hai space. Hai officials would like to consult you about possible enlistment."
+	priority
+	to offer
+		has "Enlisted by Hai [1]: done"
+	to complete
+		never
+	to fail
+		has "enlisted"
+	source
+		government "Hai"
+	destination "Hai-home"
+	on offer
+		conversation
+			`As soon as you depart your ship you discover an envoy of sorts awaiting you. "Few human visitors demonstrate care for the lasting peace we have established so long ago," the Hai begins hastily. "You are a prominent example of your kind, and we request audience on <destination>. The unfettered humans have revealed a tentativeness to prey on those who follow a path unlike their own."`
+			choice
+				`	"I suppose I can hear you out."`
+				`	"I would prefer to not get caught up in any trouble."`
+					defer
```

Comment:
Should probably be decline rather than defer, since the displayed conversation isn't quite "bland" (like the unfettered JD sale convo). If you intend to keep defer, the text should probably reflect that this is a temporary dismissal, e.g. something like, "can it wait? I'm a little busy at the moment."

Edit: I see that the intention is for decline to happen in the next mission, so I think this conversation needs to be made more agreeable for being displayed a handful of times.
Perhaps it also makes sense to include a timer of sorts on defer, so that it is not offered every day. You can use an event, or do some algebra with the day, month, and year conditions.

Reply:
Could easily change it back to `decline`, `defer` was used originally just so that the player doesn't get locked out of the shield beetle unlock forever, but I think the mission is noteworthy enough as a *don't throw it away, it sounds important* mission.